src/bun.zig

```diff
@@ -768,6 +768,22 @@ pub fn openDirAbsolute(path_: []const u8) !std.fs.Dir {
     }
 }
 pub const MimallocArena = @import("./mimalloc_arena.zig").Arena;
+pub fn getRuntimeFeatureFlag(comptime flag: [:0]const u8) bool {
+    return struct {
+        const flag_ = flag;
+        var is_enabled: ?bool = null;
+        pub fn get() bool {
+            return is_enabled orelse brk: {
+                is_enabled = false;
+                if (getenvZ(flag_)) |val| {
+                    is_enabled = strings.eqlComptime(val, "1") or strings.eqlComptime(val, "true");
+                }
+
+                break :brk is_enabled.?;
+            };
+        }
+    }.get();
+}
```

Comment:
this has a data race in multi-threaded code, `is_enabled` should be atomic

Reply:
its harmless if it's checked multiple times

Reply:
if it's null and two threads try to initialize it at the same time you get undefined behavior because the two writes do not synchronize

Reply:
In the case that two threads try to initialize it, would it matter since they would both compute the same value sincee env is constant? I guess it would result in a slight performance degradation

I'll make it atomic just to be safe anyway

Reply:
yes it would matter, two threads writing to the same memory without synchronization is undefined behavior, no matter if they're writing the same value or not